tests-legacy/Integration/PrestaShopBundle/Controller/Api/StockManagementControllerTest.php

```diff
@@ -212,7 +212,7 @@ private function assertResponseHasTotalPages($parameters, $expectedTotalPages)
         /** @var \Symfony\Component\HttpFoundation\ResponseHeaderBag $headers */
         $headers = $response->headers;
         $this->assertTrue($headers->has('Total-Pages'), 'The response headers should contain the total pages.');
-        $this->assertSame(
+        $this->assertEquals(
             $expectedTotalPages,
```

Comment:
Maybe you should change `$expectedTotalPages` instead

Reply:
The problem is this value came from Symfony, we shouldn't test it.
Since we are working with number, we don't care if Symfony returns `3` or `'3'`
We already have this debate for the 1.7.7.x  https://github.com/PrestaShop/PrestaShop/blob/1.7.7.x/tests-legacy/Integration/PrestaShopBundle/Controller/Api/StockManagementControllerTest.php#L215